USER_MANUAL.md

```diff
@@ -1874,7 +1874,7 @@ This parameter can also be set in the following file:
 
 - [Site type file](#method_site_deployment-site-type-file)
 
-_TODO_ update description more when the scale up feature is implemented.
+**Note:** If site deployment is set to False for a site/site(s) in the sites file for all non follow-up methods that make up a program, that site will functionally not be measured by the program. In this case, estimated emissions are then computed using the annual average of all estimated emissions from all measured sites of the same type or if that is not possible, the annual average of all measured sites.
```

Comment:
Nit: If site deployment is set to False for a site/site(s) in the sites file for all non-follow-up methods that make up a program, that site will be functionally not measured by the program. In this case, estimated emissions are computed using the annual average of all estimated emissions from all measured sites of the same type or, if not possible, the annual average of all measured sites.

Reply:
Can you also add this note in the [Site type file](#method_site_deployment-site-type-file) description?